app/views/course/assessment/question/scribing/new.html.slim

```diff
@@ -1,3 +1,4 @@
 - add_breadcrumb :new
+- template = 'course/assessment/question/scribing/scribing_question.json.jbuilder'
 = page_header
-div#scribing-question
+div#scribing-question data=(render partial: template)
```

Comment:
Oh no! It's remnants from my first implementation, forgot to remove it! Since editing the scribing question is already making an ajax call, might as well do the same thing for creating a new question.